src/TodoList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'; 
+  
+
+const todoList =[ 
+  { 
+    id:1,
+    title:"Take out the dog", 
+  }, 
+
+  { 
+    id:2, 
+    title:"Feed the dog",
+  }, 
+
+  { 
+    id:3, 
+    title:"Go to the store"
+  }, 
+
+  {
+    id:4, 
+    title:"Pickup dry cleaning"
+  }, 
+
+  { 
+    id:5,
+    title:"Exercise"
+  }
+]
+
+
+const TodoList = () =>{ 
+    return( 
+      
```

Comment:
nit: these newlines are unnecessary. Might be a little bit more concise to remove them.